library/Ivoz/Ast/Domain/Service/PsIdentify/UpdateByFriend.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Ivoz\Ast\Domain\Service\PsIdentify;
+
+use Ivoz\Ast\Domain\Model\PsIdentify\PsIdentifyDto;
+use Ivoz\Core\Application\Service\EntityTools;
+use Ivoz\Provider\Domain\Model\Friend\FriendInterface;
+use Ivoz\Provider\Domain\Service\Friend\FriendLifecycleEventHandlerInterface;
+
+class UpdateByFriend implements FriendLifecycleEventHandlerInterface
+{
+    /**
+     * @var EntityTools
+     */
+    protected $entityTools;
+
+    /**
+     * UpdateByFriend constructor.
+     * @param EntityTools $entityTools
+     */
+    public function __construct(
+        EntityTools $entityTools
+    ) {
+        $this->entityTools = $entityTools;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            self::EVENT_POST_PERSIST => 20
```

Comment:
self::PRIORITY_NORMAL instead of '20' would be better

Reply:
Updated in the 4 services